department-of-veterans-affairs/caseflow
# Allow CSS_ID in case assignment URL

Resolves #13433 

### Description
allow CSS_ID in URL that accesses Case Assignment page (e.g., `http://localhost:3000/queue/BVAAABSHIRE/assign`).

### Acceptance Criteria
- [ ] Case Assignment page contents are unchanged; e.g.,`http://localhost:3000/queue/BVAAABSHIRE/assign` contents is the same as `http://localhost:3000/queue/3/assign`

### Testing Plan
1. Check that `http://localhost:3000/queue/BVAAABSHIRE/assign` contents is the same as `http://localhost:3000/queue/3/assign`
1. Make sure other pages still work without any error.
